### PR TITLE
dev-tex/biber: version dump to 2.16

### PR DESCRIPTION
**Pull request by j-licht**

### CODE REVIEW

**Comment by j-licht** [comment]
I'm not quite sure why the CI reports, that this PR is broken.

**Comment by thesamesam** [comment]
Please rebase on master and force push, the tree was broken at the time - sorry!

File: perl-core/Unicode-Collate/Unicode-Collate-1.290.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline perl-core/Unicode-Collate/Unicode-Collate-1.290.0.ebuild:4]: *perl-module.eclass* claims it supports EAPI-7.

**Comment by j-licht** [comment]
> I don't know much about perl, but these packages seemed to install fine. biber-2.16 fails test phase for me.

What error did you get? Maybe I missed a test dependency.

**Comment by juippis** [comment]
https://dev.gentoo.org/~juippis/logs/biber-2.16-20210309-131435.log

**Comment by j-licht** [comment]
> https://dev.gentoo.org/~juippis/logs/biber-2.16-20210309-131435.log

The missing perl module comes from dev-perl/Unicode-LineBreak.
In the configure step biber also mentioned you are missing requierments:
```
Checking prerequisites...
  requires:
    !  IO::String is not installed
    !  Unicode::GCString is not installed
    !  Unicode::LineBreak is not installed
```
Hm I think I have to add dev-perl/IO-String as a dependency for biber.

File: virtual/perl-Unicode-Collate/perl-Unicode-Collate-1.290.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Virtual for ${PN#perl-}"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+
+RDEPEND="
+	|| ( =dev-lang/perl-5.32* =dev-lang/perl-5.30* ~perl-core/${PN#perl-}-${PV} )
```
Comment (akhuettel) [inline virtual/perl-Unicode-Collate/perl-Unicode-Collate-1.290.0.ebuild:11]: That needs fixing. This version is not provided by 5.32 or 5.30.

**Pull request closed without merging**